common/lib/xmodule/xmodule/js/src/video/05_video_quality_control.js

```diff
@@ -5,23 +5,19 @@
 define(
 'video/05_video_quality_control.js',
 [],
-function () {
+function (HtmlUtils) {
     var template = [
         '<button class="control quality-control is-hidden" aria-disabled="false" title="',
             gettext('High Definition'),
         '">',
-            '<span class="icon-fallback-img">',
-                '<span class="icon icon-hd" aria-hidden="true">HD</span>', // "HD" is treated as a proper noun
+            '<span class="icon icon-hd" aria-hidden="true">HD</span>', // "HD" is treated as a proper noun
```

Comment:
This is not caught by the linter, but was by you. :)  The gettext() calls should be escaped.  It would be best to move this to an Underscore.js template.  Otherwise, you'd have to ask @andy-armstrong what the best way to code this would be.  It may be to leave it mostly as-is, but switch the gettext() calls to `{variable}` and use HtmlUtils.interpolateHtml() to interpolate in.  The HTML would need to be wrapped with `HtmlUtils.HTML().`

I added a ticket for the linter, but don't know when anyone will get to these:
https://openedx.atlassian.net/browse/TNL-4861